app/controllers/application_controller.rb

```diff
@@ -4,4 +4,10 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   before_action :authenticate_user!
+
+  private
+
+  def after_sign_in_path_for(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.